### PR TITLE
Update Symfony packages to version 5

### PR DESCRIPTION
**Pull request by piotrbelina**
Hello,
I updated composer.json to allow to use PHPMND in Symfony 5 projects
Cheers,
Piotr

### CODE REVIEW

**Comment by piotrbelina** [comment]
Hello,
I don't know how to fix AppVeyor build
Cheers,
Piotr

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   },
   "require": {
     "php": "^7.1",
-    "symfony/console": "^4.0",
-    "symfony/finder": "^4.0",
+    "symfony/console": "^4.0 || ^5.0",
```
Comment (richardhughes) [inline composer.json]: How come your using the `||` operator here? Cant we just update it to 5.0 and then bump up the major release version?

Comment (SignpostMarv) [reply composer.json]: this PR (and this comment) is a duplicate of #102

Comment (piotrbelina) [reply composer.json]: I just learned it. #102 has title "fixes #98" which does not speak much

**Comment by kubawerlos** [comment]
@piotrbelina can we close it as a duplicate of #102?

**Comment by piotrbelina** [comment]
@kubawerlos please merge just any pull request just update the project to symfony 5

**Comment by kubawerlos** [comment]
@piotrbelina if I could :( I've started reviewing this, so maybe this will @povils merge it faster.

**Comment by aaajeetee** [comment]
I think this PR is now duplicate because #105 is merged. Correct me if I'm wrong.

**Comment by piotrbelina** [comment]
> I think this PR is now duplicate because #105 is merged. Correct me if I'm wrong.

Yes, it is a duplicate now. We can close it.

**Pull request closed without merging**